### ISSUE TITLE
Warning! Query:

### ISSUE THREAD

**Issue by RedOrion** [sealemar/BingRewards#189]
It worked fine the first couple of days and then all of a sudden I get this every time.

```
2016-04-27 23:11:37.9 -  3/36 - Search: chile
Warning! Query:
	chile
returned no results, check error_20160427-231138.624861.html file for more information
2016-04-27 23:11:56.7 -  4/36 - Search: belgrade
Warning! Query:
	belgrade
returned no results, check error_20160427-231157.328709.html file for more information
2016-04-27 23:12:14.8 -  5/36 - Search: continental army
Warning! Query:
	continental army
returned no results, check error_20160427-231215.784297.html file for more information
2016-04-27 23:12:31.8 -  6/36 - Search: erik keedus
Warning! Query:
```

**Comment by blakekl** [comment]
I'm experiencing this too.

**Comment by amayer5125** [comment]
Is anyone still experiencing this issue? I havnt gotten this errror for a while.

**Comment by blakekl** [comment]
I'm not experiencing this anymore. Now, just one of my accounts is only earning 3 points a day, but that is a different issue.

**Comment by amayer5125** [comment]
My accounts go through that from time to time. It should pass

**Comment by blakekl** [comment]
This one account has been doing it for 6-8 weeks I think.

**Comment by amayer5125** [comment]
Try updating to the newest version. We have fixed a couple issues over the past week.

**Comment by blakekl** [comment]
I run a pull before the script every day.

**Comment by amayer5125** [comment]
How long ago did you run the pull & script? We just fixed #185 4 hours ago. The people that were effected by this only earned 3 points per day.

**Comment by blakekl** [comment]
Oh cool. It usually runs early in the morning. I'll check again tomorrow. Thanks.
